### PR TITLE
>: add file emptying example

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/greater-than.md [hunk 1]
```diff
@@ -18,3 +18,7 @@
 - Redirect both `stdout` and `stderr` to `/dev/null` to keep the terminal output clean:
 
 `{{command}} &> /dev/null`
+
+- Ensure that a file is empty:
```
Comment (spageektti) [inline pages/common/greater-than.md:22]:
```suggestion
- Clear the file contents:
```
It is more clear to me what it does that way

Comment (Managor) [reply pages/common/greater-than.md:22]: I don't it's good enough. I considered that, but this command also allows creating a file. Maybe my description isn't good enough either.

Comment (spageektti) [reply pages/common/greater-than.md:22]: What about the updated one @managor?

Comment (Managor) [reply pages/common/greater-than.md:22]: I was first talking about your suggestion and then I was talking about mine.

Comment (spageektti) [reply pages/common/greater-than.md:22]: I mean I updated by suggestion @managor

Comment (Managor) [reply pages/common/greater-than.md]: Oh I see. Yes, It's good enough. I wanted a more compressed description but I guess it's impossible.

**Review by spageektti — APPROVED** [review]

**Pull request merged**